### ISSUE TITLE
WOQL query fails in 10.0.10, 10.0.12 and dev docker images

### ISSUE THREAD

**Issue by hoijnet** [terminusdb/terminusdb#816]
**Describe the bug**
A previously working query no longer works in 10.0.10. Suspect it's either in handling read_object, or how the schema elements are expressed internally.

**To Reproduce**
1. Run query 
2. TerminusX and Docker both return an error

```
const type = "MySubdocumentType";
WOQL.select(
  "v:document",
  WOQL.and(
    WOQL.eq("v:subDocType", "@schema:" + type),
    WOQL.quad("v:subDocType", "sys:subdocument", "rdf:nil", "schema"),
    WOQL.triple("v:doc", "v:subDocType", "v:c"),
    WOQL.read_object("v:doc", "v:document")
  )
)
```

The exact request rendering the error below
```
{"query":{"@type":"Select","variables":["document"],"query":{"@type":"And","and":[{"@type":"Equals","left":{"@type":"Value","variable":"subDocType"},"right":{"@type":"Value","node":"@schema:ci50"}},{"@type":"Triple","subject":{"@type":"NodeValue","variable":"subDocType"},"predicate":{"@type":"NodeValue","node":"sys:subdocument"},"object":{"@type":"Value","node":"rdf:nil"},"graph":"schema"},{"@type":"Triple","subject":{"@type":"NodeValue","variable":"doc"},"predicate":{"@type":"NodeValue","variable":"subDocType"},"object":{"@type":"Value","variable":"c"}},{"@type":"ReadObject","document_uri":{"@type":"Value","variable":"doc"},"document":{"@type":"Value","variable":"document"}}]}}}
```

`ci50` is a subdocument type.

**Expected behavior**
The query for a type being a subdocument should return documents that links to it in `read_object` format.

**Screenshots**
```
{
  "@type":"api:WoqlErrorResponse",
  "api:error": {
    "@type":"vio:WOQLSyntaxError",
    "vio:path": ["and", 3, "query" ],
    "vio:query": {
      "@type":"ReadObject",
      "document": {"@type":"Value", "variable":"document"},
      "document_uri": {"@type":"Value", "variable":"doc"}
    }
  },
  "api:message":"Not well formed WOQL JSON-LD",
  "api:status":"api:failure"
}
```

None yet, reproducing on my end. See discord for some additional context

**Info (please complete the following information):**
 - OS: MacOS with Edge Chromium
 - TerminusX and Docker 10.0.10

**Additional context**

See discord.
Hope to share additional as I continue looking into it beginning of week. Just wanted you to be aware early.

**Comment by hoijnet** [comment]
Expect to follow up with more detail during the week.

**Comment by hoijnet** [comment]
More context added. Let me know if more is needed to dive into the details

**Comment by hoijnet** [comment]
Verified in the latest images too.

**Comment by GavinMendelGleason** [comment]
Will try to sort this out today.

**Comment by GavinMendelGleason** [comment]
This is directly related to https://github.com/terminusdb/terminusdb-client-js/issues/78

**Comment by spl** [comment]
We're currently working on fixing this in the JavaScript client. Thanks!

**Comment by hoijnet** [comment]
Awesome 🙏

**Comment by spl** [comment]
This should be available with the JavaScript client release [v10.0.2](https://github.com/terminusdb/terminusdb-client-js/releases/tag/v10.0.2).

**Comment by hoijnet** [comment]
Great, thanks!! Will switch over to npm again!
